### ISSUE TITLE
Understanding the Reconstruction loss term in STORN

### ISSUE THREAD

**Issue by rppravin** [XiaoyuBIE1994/DVAE#8]
STORN has two loss terms: Reconstruction loss and KL-divergence.

Could you please explain the reconstruction loss term:
**recon = torch.sum(  x/recon_x - torch.log(x/recon_x) - 1 )**; where x is the input and recon_x is the reconstruction.

Thank you.

**Comment by XiaoyuBIE1994** [comment]
Because the speech spectrum is model by a complex Gaussian circularly symmetric distribution, so the reconstruction loss should be an Itakura-Saito Divergence. You could refer to this paper for more details:
"Notes on the use of variational autoencoders for speech and audio spectrogram modeling", Laurent Girin et al. [https://hal.archives-ouvertes.fr/hal-02349385/document](https://hal.archives-ouvertes.fr/hal-02349385/document)
